concent_api/verifier/tasks.py

```diff
@@ -90,7 +95,7 @@ def blender_verification_order(
                 cluster_response,
```

Comment:
Will it crash if the file being downloaded already exists? It should.

Any file operation where the file being created is not supposed to exist but it does should crash the verifier so that we can detect and fix the issue.

Reply:
Answered in #441